### PR TITLE
Update api to v3

### PR DESCRIPTION
**Pull request by miguelstorani**
closes  https://github.com/southxchange/market/issues/36

### CODE REVIEW

File: SouthXchange/Model/DestinationType.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace SouthXchange.Model
+{
+    public enum DestinationType
+    {
+        CryptoAddress = 0,
+        LightningNetworkinvoice = 1,
```
Comment (danielmangia) [inline SouthXchange/Model/DestinationType.cs]: LightningNetworkInvoice

File: SouthXchange/Model/DestinationType.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        SouthXchangeUserEmail = 2,
```
Comment (danielmangia) [inline SouthXchange/Model/DestinationType.cs]: UserEmailAddress

**Comment by danielmangia** [comment]
Implement public /fees endpoint as well

**Comment by danielmangia** [comment]
In method SxcContext.WithdrawAsync(string, string, decimal), use Destination/DestinationType properties instead of Address

**Comment by miguelstorani** [comment]
> In method SxcContext.WithdrawAsync(string, string, decimal), use Destination/DestinationType properties instead of Address

We should inform in the release that this is a breaking change, or leave the method as it was (keeping in mind current user's implementations) and encourage users to use `SxcContext.WithdrawAsync(WithdrawRequest)`, creating their own `WithdrawRequest`.

**Pull request merged**